dbms/src/Storages/KVStore/Read/LearnerRead.cpp

```diff
@@ -79,6 +79,7 @@ void validateQueryInfo(
 {
     RegionException::UnavailableRegions fail_region_ids;
     RegionException::RegionReadStatus fail_status = RegionException::RegionReadStatus::OK;
+    std::string fail_extra_msg;
```

Comment:
I think "err_msg" or "error_msg" is more native speaking in English

Reply:
The previous variables are all begin with `fail`